src/go/k8s/apis/redpanda/v1alpha1/cluster_webhook_test.go

```diff
@@ -544,7 +544,7 @@ func TestValidateUpdate_NoError(t *testing.T) {
 	})
 }
 
```

Comment:
It's still needed because I copied the same struct in the schema registry test, although I modified the ports. 

I think it's ok for tests. Creating some helper function for this might be too much IMO 🤔